corehq/apps/app_manager/detail_screen.py

```diff
@@ -102,12 +102,13 @@ def __init__(self, app, module, detail, column, sort_element=None,
 
     @property
     def locale_id(self):
-        if not is_sort_only_column(self.column):
+        if is_sort_only_column(self.column):
+            sort_element = [sort_element for sort_element in self.detail.sort_elements if sort_element.field == self.column.field][0]
```

Comment:
Will check about `SortOnlyDetailColumn`. Not sure about skipping the sort element since thats needed for the field value and display value for now. Will see how things change.